api/api.yml

```diff
@@ -112,7 +165,7 @@ paths:
       responses:
         '201':
           description: Application added
-  /launcher/app-user/applications/{id}:
+  /launcher/app-user/application/{id}:
     delete:
         tags:
```

Comment:
Dalej nie wiem, po co to tu zostawiać.

Reply:
Mogę usunąć, ale może też zostać. Nie ma obowiązku korzystania :)